### DISCUSSION TITLE
Is there a way to analyze only one asset?

### DISCUSSION THREAD

**Discussion by frank01100110** [ssantoshp/Empyrial#81] [General]
Is there a way to analyze only one asset?

**Comment by santoshlite** [comment]
Yes but you have to give more than one stock (I need to fix this). Here is the code:
```py
portfolio = Engine(    
                  start_date= "2018-06-09", #start date for the backtesting
                  portfolio= ["BABA", "TSLA"], #assets in your portfolio 
                  weights = [0, 1], #equal weighting is set by default
                  benchmark = ["SPY"] #SPY is set by default
)

empyrial(portfolio)
```

**Comment by frank01100110** [comment]
> Yes but you have to give more than one stock (I need to fix this). Here is the code:
> ```py
> portfolio = Engine(    
>                   start_date= "2018-06-09", #start date for the backtesting
>                   portfolio= ["BABA", "TSLA"], #assets in your portfolio 
>                   weights = [0, 1], #equal weighting is set by default
>                   benchmark = ["SPY"] #SPY is set by default
> )
> 
> empyrial(portfolio)
> ```
> 

Thank you
